lib/opencensus/stats/measure.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+
+require "opencensus/stats/measurement"
+
+module OpenCensus
+  module Stats
+    # Measure
+    #
+    # The definition of the Measurement.
+    class Measure
+      # Describes the unit used for the Measure.
+      # Should follows the format described by
+      # http://unitsofmeasure.org/ucum.html
+      # Unit name for general counts
+      # @return [String]
+      UNIT_NONE = "1".freeze
+
+      # Unit name for bytes
+      # @return [String]
+      BYTE = "By".freeze
+
+      # Unit name for Kilobytes
+      # @return [String]
+      KBYTE = "kb".freeze
+
+      # Unit name for Seconds
+      # @return [String]
+      SEC = "s".freeze
+
+      # Unit name for Micro seconds
+      # @return [String]
+      MS = "ms".freeze
```

Comment:
Are you sure this is correct for microseconds? I'd expect "us" rather than "ms". Or is this supposed to be milliseconds?

Reply:
Fixed